runtime/reconcilers/report.go

```diff
@@ -301,10 +302,11 @@ func (r *ReportReconciler) sendReport(ctx context.Context, self *runtimev1.Resou
 	exportURL.RawQuery = exportURLQry.Encode()
 
 	sent := false
-	for _, notifier := range rep.Spec.NotifySpec.Notifiers {
-		switch notifier.Spec.(type) {
-		case *runtimev1.NotifierSpec_Email:
-			for _, recipient := range notifier.GetEmail().Recipients {
+	for _, notifier := range rep.Spec.Notifiers {
+		switch notifier.Connector {
+		case "email":
+			recipients := pbutil.ToSliceString(notifier.Properties.AsMap()["recipients"].([]any))
```

Comment:
Would also suggest a `val, ok := ...` type assertion here to keep it safe from panics